helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -106,6 +106,17 @@ splunkPlatform:
     #   requests_per_second is the average number of requests per seconds.
     queueSize: 5000
 
+    # When enabled, it uses file_storage extension to persist the queue data.
+    # This can be used as an alternative redundancy mechanism for data being exported.
```

Comment:
```suggestion
    # This option enables the persistent queue to store data on the disk instead of memory before sending it to the backend.
    # It allows setting higher queue limits and preserving the data across restarts of the collector container.
```